### PR TITLE
Add render tests for LoginFindSchool

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-13534
affects: none
release note: none

### CODE REVIEW

**Comment by instructure-cx** [comment]
Scan submitted to Checkmarx

**Comment by instructure-cx** [comment]
### Checkmarx scan completed
[Full Scan Details](https://cxmgr.inssec.info/CxWebClient/ViewerMain.aspx?scanid=1014145&projectid=17)
#### Checkmarx Scan Summary
Severity|Count
---|---
High|0
Medium|0
Low|0
Informational|0

#### Violation Summary
Severity|Count
---|---

File: Core/Core/Login/LoginFindSchoolViewController.swift [hunk 1]
```diff
@@ -18,17 +18,21 @@
...
-class LoginFindSchoolViewController: UIViewController, LoginFindSchoolViewProtocol {
-    @IBOutlet weak var keyboardSpace: NSLayoutConstraint?
-    @IBOutlet weak var loadingView: UIActivityIndicatorView?
-    @IBOutlet weak var promptLabel: DynamicLabel?
-    @IBOutlet weak var resultsTableView: UITableView?
-    @IBOutlet weak var searchField: UITextField?
+class LoginFindSchoolViewController: UIViewController {
+    @IBOutlet weak var keyboardSpace: NSLayoutConstraint!
+    @IBOutlet weak var loadingView: UIActivityIndicatorView!
+    @IBOutlet weak var promptLabel: UILabel!
```
Comment (bluwave) [inline Core/Core/Login/LoginFindSchoolViewController.swift]: is this in the right font now?

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Build failed, skipping coverage check
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: None

[MBL-13534](https://instructure.atlassian.net/browse/MBL-13534)

<details><summary>:x: Swift lint</summary>


    ⚠️  /Users/vagrant/git/Core/CoreTests/Login/LoginFindSchoolViewControllerTests.swift:27:1: Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)
    Linting 'AnalyticsTests.swift' (484/615)
    Linting 'RichContentEditorViewControllerTests.swift' (485/615)
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Login/LoginFindSchoolViewController.swift (hunk 1 above)
Comment (vanwagonet) [reply Core/Core/Login/LoginFindSchoolViewController.swift]: It still is a `DynamicLabel` in the storyboard with the font set up there. The controller doesn't need to care though. The extras added by `DynamicLabel` are only used by storyboards.

Comment (bluwave) [reply Core/Core/Login/LoginFindSchoolViewController.swift]: cool cool cool

**Pull request merged**